app/src/main/java/org/mozilla/focus/FocusApplication.java

```diff
@@ -31,17 +31,21 @@ public void onCreate() {
     }
 
     private void enableStrictMode() {
-        if (AppConstants.isReleaseBuild()) {
+        // Android/WebView sometimes commit strict mode violations, see e.g.
+        // https://github.com/mozilla-mobile/focus-android/issues/660
+        if (AppConstants.isReleaseBuild() || AppConstants.isBetaBuild()) {
             return;
         }
 
         StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder()
                 .detectAll()
+                .penaltyLog()
                 .penaltyDeath()
```

Comment:
I added the penaltyLog() mainly to enable logs (penaltyDeath() by itself doesn't print a stack trace, at least on the devices I've been using, but penaltyDeath+penaltyLog means the app dies and there's a visible trace to find what the violation was). (I'll update my patches with the new policies!)